apps/frontend/src/libs/components/activity-chart/activity-chart.tsx

```diff
@@ -0,0 +1,22 @@
+import { Line, LineChart } from "recharts";
+
+import { type ActivityChartData } from "./libs/types/types.js";
+
+type Properties = {
+	data: ActivityChartData;
+};
+
+const ActivityChart = ({ data }: Properties): JSX.Element => {
+	return (
+		<LineChart data={data} height={40} width={120}>
+			<Line
+				dataKey="commitsNumber"
+				dot={false}
+				stroke="var(--color-brand-primary)"
```

Comment:
can't we pass this using css className

Reply:
I can do it this way
![image](https://github.com/user-attachments/assets/e4d9d85d-a9ad-4cd9-80bb-352e5b040abe)
![image](https://github.com/user-attachments/assets/00f696d7-7d01-4c95-85bd-e336bdacb48c)
But it seems to me, that this violates B10 QC

Reply:
To override library styles it's ok

Reply:
Done